src/content/docs/en/setup/requirements.mdx

```diff
@@ -11,7 +11,7 @@ These are our suggestions for the most basic (daily-driveable) system:
 
 - 4GB of RAM (8GB or more recommended)
 - 64-bit processor (x86_64/amd64)
-- 16GB of free disk space (24GB or more recommended)
+- 20GB of free disk space (32GB or more recommended)
```

Comment:
?

Reply:
it's getting harder and harder to find 16gb drives and the experience is really really painful.

Reply:
that's not the point, if I can boot UM with 16G storage space then that's the minimum

Reply:
we do say suggestions and daily driveable

Reply:
Alright then, fine whatever, I guess you're right